stacks/api/bucket/scheduler.ts

```diff
@@ -39,7 +39,8 @@ export class DocumentScheduler {
 
     const publish = () => {
       delete data._schedule;
-      this.bds.replaceOne(bucket, data);
+      data._id;
```

Comment:
Is this line needed ?

Reply:
It should have been `delete data._id`. Fixed on last commit.